src/libaktualizr/uptane/imagerepository.cc

```diff
@@ -173,6 +173,12 @@ void ImageRepository::verifyTargets(const std::string& targets_raw, bool prefetc
     if (targets->version() != snapshot.role_version(Uptane::Role::Targets())) {
       throw Uptane::VersionMismatch(RepositoryType::IMAGE, Uptane::Role::TARGETS);
     }
+  } catch (const Uptane::SecurityException& e) {
+    if (hash_change_expected)
+      LOG_INFO << "Signature verification for Image repo Targets metadata failed";
+    else
+      LOG_ERROR << "Signature verification for Image repo Targets metadata failed";
```

Comment:
Maybe we should add `e.what()` to the error message too?

Reply:
I don't see a problem with that, makes sense. Adding.